### PR TITLE
add touch bar tokens mac

### PR DESCRIPTION
**Pull request by NotNestor**
Add colour tokens for touch bar on mac

https://www.figma.com/file/H19CutixoN8SJGO5EXwcc8/Components---MacOS?node-id=15991%3A144836
https://www.figma.com/file/uTiF8j59yEbhbT3pgLYgxs/Tokens---Theme-Color---Additional?node-id=1302%3A979

Screenshot 2022-06-22 at 13 46 20

### CODE REVIEW

**Comment by NotNestor** [comment]
theme tokens need to be defined for the core tokens before this PR can be progressed

File: platformcomponents/desktop/touchbar.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "figma": "https://www.figma.com/file/H19CutixoN8SJGO5EXwcc8/Components---MacOS?node-id=15991%3A144836",
+    "background": {
+      "messages": "@theme-common-touchbar-background-blue",
+      "contacts": "@theme-common-touchbar-background-mint",
+      "content": "@theme-common-touchbar-background-pink",
+      "call": "@theme-common-touchbar-background-green",
+      "whiteboard": "@theme-common-touchbar-background-purple",
+      "meetings": "@theme-common-touchbar-background-orange",
+      "links": "@theme-common-touchbar-background-violet",
+      "encCall": "@theme-common-touchbar-background-red",
```
Comment (nataliadelmar) [inline platformcomponents/desktop/touchbar.json:13]: should it be endCall?

File: theme-data/win-system-hc/win-system-hc-common.json [hunk 2]
```diff
@@ -569,7 +587,7 @@
         "primary": {
           "disabled": "@color-hc-SystemColorGrayTextColor"
         }
-      }
+      }
```
Comment (nataliadelmar) [inline theme-data/win-system-hc/win-system-hc-common.json:590]: did you add this space in purpose? if not, please delete :)

**Review by nataliadelmar — CHANGES_REQUESTED** [review]
looks good to me. Just a typo and that's it

File: platformcomponents/desktop/touchbar.json (hunk 1 above)
Comment (NotNestor) [reply platformcomponents/desktop/touchbar.json:13]: 👍

**Pull request merged**